clarity/src/vm/callables.rs

```diff
@@ -257,6 +257,19 @@ impl DefinedFunction {
     pub fn get_identifier(&self) -> FunctionIdentifier {
         self.identifier.clone()
     }
+
+    pub fn get_arguments(&self) -> &Vec<ClarityName> {
+        &self.arguments
+    }
+
+    pub fn get_arg_types(&self) -> &Vec<TypeSignature> {
+        &self.arg_types
+    }
+
+    #[cfg(feature = "developer-mode")]
```

Comment:
Is there a reason this is feature-gated?  It's not like omitting it is going to save considerable space in the binary ;)

Reply:
Because in representations.rs, the field is gated:
```rust
pub struct SymbolicExpression {
    pub expr: SymbolicExpressionType,
    // this id field is used by compiler passes to store information in
    //  maps.
    // first pass       -> fill out unique ids
    // ...typing passes -> store information in hashmap according to id.
    //
    // this is a fairly standard technique in compiler passes
    pub id: u64,

    #[cfg(feature = "developer-mode")]
    pub span: Span,
}
```